POSTINSTALL-README.md

```diff
@@ -2,7 +2,8 @@
 
 After rebasing to secureblue, the following steps are recommended.
 
-## Set a GRUB password
+## GRUB
+### Set a password
```

Comment:
We should add some text between empty headers

Reply:
I think that is a matter of taste? You could do that, but its not mandatory